storage/blob_test.go

```diff
@@ -269,6 +269,30 @@ func (s *StorageBlobSuite) TestBlobCopy(c *chk.C) {
 	c.Assert(b, chk.DeepEquals, body)
 }
 
+func (s *StorageBlobSuite) TestStartBlobCopy(c *chk.C) {
+	if testing.Short() {
+		c.Skip("skipping blob copy in short mode, no SLA on async operation")
+	}
+
+	cli := getBlobClient(c)
+	cnt := randContainer()
+	src := randName(5)
+	dst := randName(5)
+	body := []byte(randString(1024))
+
+	c.Assert(cli.CreateContainer(cnt, ContainerAccessTypePrivate), chk.IsNil)
+	defer cli.deleteContainer(cnt)
+
+	c.Assert(cli.putSingleBlockBlob(cnt, src, body), chk.IsNil)
+	defer cli.DeleteBlob(cnt, src, nil)
+
+	// given we dont know when it will start, can we even test destination creation?
+	// will just test that an error wasn't thrown for now.
```

Comment:
Try to test with `BlobExists`.
To test aborting maybe you'll need to copy a bigger blob, so it actually isn't copied so fast that it cannot be aborted.